Gemfile.lock

```diff
@@ -105,11 +105,11 @@ GEM
     mime-types-data (3.2016.0521)
     mini_portile2 (2.3.0)
     minitest (5.10.2)
-    mongo (2.1.2)
-      bson (~> 3.0)
-    mongoid (5.0.0)
-      activemodel (~> 4.0)
-      mongo (~> 2.1)
+    mongo (2.5.3)
+      bson (>= 4.3.0, < 5.0.0)
+    mongoid (7.0.5)
+      activemodel (>= 5.1, < 6.1)
```

Comment:
This requirement is not currently being met, also it seems a few other gem versions depend on mongoid < 7.  We may need to upgrade those as well.

Reply:
@bderusha Upgrading activemodel to 5.1 or greater breaks another dependency protected_attributes which is only compatible with activemodel < 5. So I don't think we can upgrade to mongoid 7.0.5 as it requires activemodel > 5.1 (which is not compatible with protected_attributes)